src/main/client/app/pages/stacks/stack-variable.vue

```diff
@@ -4,12 +4,20 @@
     :description="description"
   >
     <b-input
-      v-if="! isListRegex"
+      v-if="! isListRegex && ! complexType"
```

Comment:
since we have 3 cases now, what do you think about using the following structure:

```
<b-textarea v-if="complexType">...
<b-select v-else-if="isListRegex">...
<b-input v-else>...
```

Reply:
yes ! done in next commit